src/components/Search/SearchOutputTable.vue

```diff
@@ -1,21 +1,18 @@
 <template>
-    <table class="table">
+    <table class="table table-hover">
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces but found 4 spaces.](https://app.codacy.com/gh/FAIRsharing/fairsharing.github.io/pullRequest?prid=4893952)